### PR TITLE
Feature/product tile

### PR DESCRIPTION
**Pull request by jess-lam**

### CODE REVIEW

File: src/components/Product/ProductGrid.tsx [hunk 1]
```diff
@@ -11,7 +11,23 @@ const ProductGrid: React.FC<Props> = ({ products }) => {
     <div className={styles.product__grid__container}>
       <div className={styles.flex__row}>
         {products.map((product: IProduct) => {
-          return <p key={product.id}>{product.title}</p>;
+          return (
+            <div className={styles.product_tile} key={product.id}>
```
Comment (gilwong00) [inline src/components/Product/ProductGrid.tsx:15]: this should be its own component

**Review by gilwong00 — CHANGES_REQUESTED** [review]
@jess-lam we should make the product tile its own component. can you make a new .tsx file and a scss file for the product tile inside the product folder

File: src/components/Product/ProductGrid.tsx (hunk 1 above)
Comment (jess-lam) [reply src/components/Product/ProductGrid.tsx]: Ok. I just extracted it out.

File: src/components/Product/ProductGrid.module.scss [hunk 2]
```diff
@@ -17,10 +17,83 @@
     flex-direction: column;
   }
 
-  // replace this with product tile component
-  p {
-    flex: 1 1 33%;
-    padding: 20px;
-    height: 100px;
-  }
+  // .product_tile {
```
Comment (gilwong00) [inline src/components/Product/ProductGrid.module.scss:20]: can we remove these comments if they are no longer needed

File: src/components/Product/ProductGrid.tsx [hunk 3]
```diff
@@ -11,7 +12,9 @@ const ProductGrid: React.FC<Props> = ({ products }) => {
     <div className={styles.product__grid__container}>
       <div className={styles.flex__row}>
         {products.map((product: IProduct) => {
-          return <p key={product.id}>{product.title}</p>;
+          return (
+            <ProductTile product={product} />
```
Comment (gilwong00) [inline src/components/Product/ProductGrid.tsx:16]: need a key here

File: src/components/ProductTile/ProductTile.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  product: IProduct;
+}
+
+const ProductTile: React.FC<Props> = ({ product }) => {
+  return (
+    <div className={styles.product_tile} key={product.id}>
+    <div className={styles.product_image}>
+      <img className={styles.image} src={product.image} />
+      <div className={styles.heart_container}>
+      <img className= {styles.heart} src="/heart.png"/>
```
Comment (gilwong00) [inline src/components/ProductTile/ProductTile.tsx:15]: i dont think we need this heart icon atm. lets comment it out

**Review by gilwong00 — CHANGES_REQUESTED** [review]

**Comment by gilwong00** [comment]
Screen Shot 2021-04-07 at 8 11 53 PM

@jess-lam couple of changes here
- lets up the font size to 16px for the title
- im not loving where the price is can we display it on the same line as the add to bag button, @courtniwong what do you think?
- lets rename add to bag to add to cart

**Pull request merged**